pkg/controller/sparkapplication/controller.go

```diff
@@ -155,13 +142,7 @@ func (c *Controller) Start(workers int, stopCh <-chan struct{}) error {
 		// the worker after one second.
 		go wait.Until(c.runWorker, time.Second, stopCh)
 	}
-
-	go c.runner.run(stopCh)
 	go c.sparkPodMonitor.run(stopCh)
```

Comment:
lets get rid of this, We should just initialize an eventhandler on the PodInformer that extracts the sparkappid and enqueues it. Lets also split out the enqueue method as separate method